integration/package.json

```diff
@@ -10,6 +10,7 @@
     "dev": "lerna run test --scope integration --parallel --include-filtered-dependencies"
   },
   "dependencies": {
+    "@bitcoinerlab/secp256k1": "^1.1.1",
```

Comment:
If you're wondering why we're using this on top of tinysecp256k1, here is a little writeup

latest bitcoinjs-lib pretty much works as-is, assuming you update everything in hdwallet to new types and runtime to match the new shape of things. The only thing that doesn't work is taproot
taproot requires initing bitcoinjs-lib with an ecc lib, other script types work fine without said init. But after spending way too many hours between trying to get latest tiny-secp256k work in web (which requires wasm support and I was able to get working there), had some issue re: one of the methods not being defined although it should've been. I'm not sure if that's an us thing, but IMO, doesn't make sense trying to spend more time on this, instead went with https://github.com/bitcoinerlab/secp256k1, which is pretty much tiny-secp256k1 but using noble-secp256k1 under the hood, allowing us to effectively use tiny-secp256k1 without the wasm support hassle, and without that weird error. Will still keep the wasm support in the matching web PR for futureproofing, and if we feel like we don't want to rely bitcoinerlab/secp256k1 as an ecc lib, happy to spend more time trying to get the regular tiny-secp256k1 working as a follow-up